testsupport/wait/host.go

```diff
@@ -563,6 +563,46 @@ func (a *HostAwaitility) WaitForTestResourcesCleanup(initialDelay time.Duration)
 				return false, nil
 			}
 		}
+
+		spaceBindingsList := &toolchainv1alpha1.SpaceBindingList{}
```

Comment:
should be named `spaceBindingList` but it's a minor comment

Reply:
fixed in [4c30f44df](https://github.com/mfrancisc/toolchain-e2e/commit/4c30f44df8198939255740d6fb574b80e1db19bf)